src/ray/rpc/server_call.h

```diff
@@ -204,6 +205,7 @@ class ServerCallImpl : public ServerCall {
   void SetState(const ServerCallState &new_state) override { state_ = new_state; }
 
   void HandleRequest() override {
+    stats_handle_ = io_service_.stats().RecordStart(call_name_);
```

Comment:
actually, is it possible to track both 

1. handler execution time
2. time to reply

? I think there are times 1 is much more important than 2

Reply:
We already have 1.

```
if (!io_service_.stopped()) {
      io_service_.post([this, auth_success] { HandleRequestImpl(auth_success); },
                       call_name_ + ".HandleRequestImpl",
```